src/Composer/Installers/EzPlatformInstaller.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Composer\Installers;
+
+class EzPlatformInstaller extends BaseInstaller
+{
+    protected $locations = array(
+        'meta-assets' => 'web/assets/ezplatform/',
+        'assets' => 'web/assets/ezplatform/{$name}/',
```

Comment:
I would prefer to keep `ezplatform` as it makes clear this is directory owned by us for a very specific purpose while `package` is more generic